terraform-plans/templates/github/charm_tox.ini.tftpl

```diff
@@ -0,0 +1,102 @@
+[tox]
```

Comment:
Please add comment saying this is a centralized managed file

Reply:
Thanks, added :)